test/executor/drop_test.cpp

```diff
@@ -219,5 +219,52 @@ TEST_F(DropTests, DroppingTrigger) {
   txn_manager.CommitTransaction(txn);
 }
 
+TEST_F(DropTests, DroppingIndexByName) {
```

Comment:
Your test is just exciting the code not checking to ensure that the index is actually deleted. You need to add EXPECT_EQ statements like the other tests to make sure that the index is actually dropped. Also you need to commit the transaction after each change and then start a new one.

Reply:
Thanks for the detailed description. I should have understood this earlier.